acceptance_tests/features/steps/view_collection_exercise_live_state.py

```diff
@@ -39,14 +38,10 @@ def go_live_date_hits(_, survey, period):
 @then('the state of a collection exercise is to be changed to Live')
 @then('they are able to see the Live Status for that Collection Exercise')
 def ce_details_state_is_live(_):
-    ce_state = collection_exercise_details.get_status()
-    for i in range(20):
-        if collection_exercise.is_live(ce_state):
-            break
-        time.sleep(1)
+    if not browser.is_text_present('Live', wait_time=2):
```

Comment:
Not sure if this will work because the page won't be updated asynchronously although the data is. It is likely it will always wait for 2 seconds rather that up to 2 seconds